src/main/webapp/WEB-INF/view/navbar.jsp

```diff
@@ -10,4 +14,7 @@
   <% } %>
   <a href="/about.jsp">About</a>
   <a href="/activityfeed">Activity Feed</a>
-</nav>
\ No newline at end of file
+  <%if(navBarUser != null && ifAdmin.getIsAdmin() == true){%>
```

Comment:
It might be cleaner to just have one variable for navBarUser and ifAdmin, say `User user` and checking if it isn't null and if it's an admin.